### PR TITLE
Pr zoey/episode buttons

### PR DESCRIPTION
**Pull request by lush-life**
新增單集資訊頁 share/add to playlist/download button
device-2020-12-07-171854
device-2020-12-07-171828

### CODE REVIEW

File: app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt [hunk 1]
```diff
@@ -170,18 +181,98 @@ private fun EpisodeProgressBar(
...
+}
+
+@Composable
+private fun AddToPlaylistButton(
+    isAdded: Boolean,
+    onClick: () -> Unit = {},
+    modifier: Modifier = Modifier
+) {
+    val (asset, tint) = if (isAdded) {
+        Icons.Filled.CheckCircle to iconHighlightTInt
```
Comment (dev-weiqi) [inline app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt]: typo?  iconHighlightTInt -> iconHighlightTint

**Review by dev-weiqi — COMMENTED** [review]
done

**Review by ivanisidrowu — APPROVED** [review]

File: app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt [hunk 2]
```diff
@@ -170,18 +181,98 @@ private fun EpisodeProgressBar(
...
+
+    ConstraintLayout(modifier = modifier) {
+        val (playBtn, shareBtn, addBtn, downloadBtn) = createRefs()
+        PlayButton(
+            isPlaying = isPlaying,
+            onClick = onClickPlay,
+            modifier = Modifier.constrainAs(playBtn) {
+                start.linkTo(parent.start)
+                top.linkTo(parent.top)
+                bottom.linkTo(parent.bottom)
```
Comment (AmyPlace) [inline app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt:210]: missing start padding

**Review by AmyPlace — COMMENTED** [review]
👀

File: app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt (hunk 2 above)
Comment (lush-life) [reply app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt:210]: Done.

File: app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt (hunk 1 above)
Comment (lush-life) [reply app/src/main/java/tw/jizah/popocast/ui/episode/EpisodeDetail.kt]: Done.

**Review by AmyPlace — APPROVED** [review]

**Review by dev-weiqi — APPROVED** [review]

**Pull request merged**